active/RoundedCorner/roundedcorner.md

```diff
@@ -0,0 +1,112 @@
+# Corner Radius
+![button](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/chigy/roundedcorner/active/RoundedCorner/ImageFiles/Button2.png)
+<Add thinking behind why we are rounding corners related to Fluent design.>
+
+## Background
+This "spec" will be what is being proposed to be copied over to docs.microsoft.com as a documentation under https://docs.microsoft.com/en-us/windows/uwp/design/style/ "Corner Radius" page to be created.
+
+I am trying out writing a little more "background explanation (WHY)" that our customers have expressed we provide with our documentatiton in some of our focus groups. I would like feedback as this does not follow normal documentation pattern.
+
+## Principles
+*Todo: Work with Fluent collective to put together something here around WHY*
+
+## Default control designs
+There are mainly 3 types of UI parts where the radii of the corners are being rounded and here are the descriptions of how this is applied to default controls.
+
+*Corners of rectangle UI elements*
+- These UI elements include basic UIs like buttons that users see them on a surface of the screen at all times.
+- The default radius value we use are 2px for these UI.
+![Button](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/chigy/roundedcorner/active/RoundedCorner/ImageFiles/Button.png)
+
+**Controls being rounded**
+- Button
+- CheckBox
+- ComboBox
+- DropDownButton
+- Slider
+- SplitButton
+- ToggleButton
+- ToggleSplitButton
+- Flipview
+- GridView
+- ListView
+- TreeView
+- ContentDialog
+- AutoSuggestBox
+- PasswordBox
+- RichEditBox
+- TextBox
+- DatePicker
+- CalendarDatePicker
+- Tab control
+
+*Corners of flyout UI elements*
+- These UI elements are those that appear temporarily on UI like a MenuFlyout or Flyout.
+- The default radius value we use here are 4px.
+![flyout](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/chigy/roundedcorner/active/RoundedCorner/ImageFiles/Flyout.png)
+
+**Controls being rounded**
+- CalendarDatePicker
+- DatePicker
+- TimePicker
+- Flyout
+- TeachingTip
+- ToolTip
+- DropDownButton
+- SplitButton
+- Slider
```

Comment:
This one is confusing, slider isn't a transient control. Are you talking about Slider's value tooltip? It may be worthwhile to call out which control part is being rounded, or at least say that this 4px radius rule doesn't necessary apply to the entire control listed.

Reply:
```suggestion
- Slider (tooltip flyout only)
```

Reply:
@YuliKl , does this make clearler?